dm: add log redaction configuration item in dm worker log file (#20663)
pingcap/docs: dm/dm-worker-configuration-file.md

- Specifies the log file directory. If this parameter is not specified, the logs are printed onto the standard output.

> ### `redact-info-log` <span class="version-mark">New in v9.0</span>

Reviewer: ```suggestion
### `redact-info-log` <span class="version-mark">New in v9.0.0</span>
```

> - Determines whether to log DM query arguments details in logs.

Reviewer: ```suggestion
- Controls whether to enable log redaction. When this configuration item is set to `true`, DM-worker logs are redacted to hide detailed DM query arguments. For more information, see [DM-worker Log Redaction](/log-redaction.md#log-redaction-in-DM-worker-side).
- Default value: `false`
- Value options: `false`, `true`
```

- Specifies the log file directory. If this parameter is not specified, the logs are printed onto the standard output.

### `redact-info-log` <span class="version-mark">New in v9.0.0</span>

> - Controls whether to enable log redaction. When this configuration item is set to `true`, DM-worker logs are redacted to hide detailed DM query arguments. For more information, see [Log redaction in DM-worker side](/log-redaction.md#log-redaction-in-DM-worker-side).

Reviewer: ```suggestion
- Controls whether to enable log redaction. When this configuration item is set to `true`, DM-worker logs are redacted to hide detailed DM query arguments. For more information, see [Log redaction in DM-worker side](/log-redaction.md#log-redaction-in-dm-worker-side).
```
